storage-bigtable/src/bigtable.rs

```diff
@@ -26,10 +26,14 @@ mod google {
 use google::bigtable::v2::*;
 
 pub type RowKey = String;
-pub type CellName = String;
-pub type CellValue = Vec<u8>;
 pub type RowData = Vec<(CellName, CellValue)>;
 pub type RowDataSlice<'a> = &'a [(CellName, CellValue)];
+pub type CellName = String;
+pub type CellValue = Vec<u8>;
+pub enum CellData<B, P> {
+    Bincode(B),
+    Protobuf(P),
```

Comment:
When we add new fields, do we need a new enum entry here or can the protobuf deserialization handle it (ie, just a .proto file addition)?

Reply:
No new enum entries, just .proto change

Reply:
Missing fields will be set to default values